config.go

```diff
@@ -67,4 +67,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Labels holds runtime-specified information that is not structured by this specification
+	Labels map[string]interface{} `json:"labels"`
```

Comment:
On Mon, Nov 16, 2015 at 02:12:19PM -0800, Vish Kannan wrote:
> > +	// Labels holds runtime-specified information that is not structured by this specification
> > +	Labels map[string]interface{} `json:"labels"`

We're not structuring it, so it should just probably be ‘interface{}’,
not ‘map[string]interface{}’ (and sorry about the slow path of
suggestions from json.RawMessage [1] to map[string]interface{} [2] to
just interface{} :/).

[1]: https://github.com/opencontainers/specs/pull/188#issuecomment-147030855
[2]: https://github.com/opencontainers/specs/pull/188#discussion_r42244153